.github/workflows/build-release.yml

```diff
@@ -77,3 +77,12 @@ jobs:
     secrets:
       registry: ${{ secrets.IMAGE_REGISTRY }}
       token: ${{ secrets.GITHUB_TOKEN }}
+
+  copy-to-rhisv:
+    needs: [build-release, build-multiarch]
```

Comment:
Should only need build-multiarch here. Why both?

Reply:
I need the inputs from build-release. The build-multiarch target does not produce outputs.

Reply:
Ah. Hadn't run into that scenario before.